app/src/main/java/com/kickstarter/libs/LakeTrackingClient.kt

```diff
@@ -51,7 +50,7 @@ class LakeTrackingClient(
         data.put("properties", propertiesJSON)
 
         val record = JSONObject()
-        record.put("partition-key", UUID.randomUUID().toString())
+        record.put("partition-key", this.loggedInUser?.id() ?: deviceDistinctId())
```

Comment:
@amyfain does it matter that this will be a number for logged in users and a string for logged out users?